src/serving/route_prediction.py

```diff
@@ -57,7 +57,11 @@ def _get_gas_station_points_near_path(path, radius=0.02):
 def predict_price(id, time):
     model, _, df_forecast = train_and_predict(gas_station_id=id, start_time=time, end_time=time, use_cached=True,
                                               cache=True)
-    deci_cent = round(Decimal(df_forecast.loc[0, 'yhat']), 0)
+    if df_forecast is None:
+        # data of gas station was not available
+        # return an extremely high value for a gas price that will not be included in our prediction
+        return 1000
```

Comment:
I think the style would be nicer if we'd return e.g. `-1` and check for negative prices before we use a price. Otherwise, we're doing some assumptions here that could backfire (What if in the future we end up with such high prices? What if we just want to predict the prices without any route prediction? [In this case, it would not be obvious at the first glance that the predicted value is invalid])

Reply:
Other suggestion: sys.maxint